packages/third-parties/package.json

```diff
@@ -33,6 +33,6 @@
   },
   "peerDependencies": {
     "next": "^13.0.0 || ^14.0.0 || ^15.0.0",
-    "react": "^18.2.0 || 19.0.0-rc-65a56d0e-20241020"
+    "react": "^18.2.0 || ^19.0.0-0"
```

Comment:
We can't do this since some package managers auto-install peer dependencies and the React version scheme for Canaries is not setup to support ordered version. 

This may install a version that's not actually the latest Canary but latest is what we'd like here.